### ISSUE TITLE
ListItem.Header changed to ListItem.Subheader?

### ISSUE THREAD

**Issue by justintime4tea** [papyros/files-app#7]
### Error:  "ListItem.Header" type not found

---

</br>
#### Issue Description:

When attempting to run **files-app** I get two errors related to ListItem.Header type
_I had to fix the first ListItem.Header reference to get subsequent error, See Error 01 and 02 below_

</br>
#### Application stdout:

```
Trying to load QML from: "/usr/local/bin/qml/main.qml"
Trying to load QML from: "/mnt/hd2/Dev/git/papyros-files-app/build/qml/main.qml"
Trying to load QML from: "/home/justin/.local/share/papyros-files/qml/main.qml"
Trying to load QML from: "/usr/local/share/papyros-files/qml/main.qml"
QQmlApplicationEngine failed to load component
```
##### ListItem.Header - Error 1

```
file:///usr/local/share/papyros-files/qml/main.qml:46 Type FolderPage unavailable
file:///usr/local/share/papyros-files/qml/FolderPage.qml:113 Type PlacesSidebar unavailable
file:///usr/local/share/papyros-files/qml/components/PlacesSidebar.qml:32 ListItem.Header is not a type
```
##### ListItem.Header - Error 2

```
file:///usr/local/share/papyros-files/qml/main.qml:46 Type FolderPage unavailable
file:///usr/local/share/papyros-files/qml/FolderPage.qml:89 Type InfoSidebar unavailable
file:///usr/local/share/papyros-files/qml/components/InfoSidebar.qml:93 ListItem.Header is not a type
```
#### Potential Fix:

Changing the single occurrence of **ListItem.Header** to **ListItem.Subheader** in both `qml/components/InfoSideBar.qml - line 93` and `qml/components/PlacesSideBar.qml - line 32` gets rid of this error.

_Fixing in my fork and submitting pull-request though not sure if this Header->Subheader change assumption on my part is accurate or if Subheader is correct for this fix._

**Comment by justintime4tea** [comment]
Extended details for earlier issue #4 

_When issue #4 is closed this issue will effectively be closed as well._
_Apologies for the duplicate bug post_

**Comment by ricardomv** [comment]
Fixed in #12
